third_party/ruby/build-ruby.bzl

```diff
@@ -451,16 +459,47 @@ _ruby_internal_headers = rule(
 def _rubyfmt_static_deps_impl(ctx):
     ruby_info = ctx.attr.ruby[RubyInfo]
 
-    libs = depset([ruby_info.static_libs])
+    libs = [ruby_info.static_lib_ruby, ruby_info.static_lib_ripper]
+    cc_toolchain = find_cpp_toolchain(ctx)
+
+    cc_toolchain = ctx.attr._cc_toolchain[cc_common.CcToolchainInfo]
+    feature_configuration = cc_common.configure_features(
+        ctx = ctx,
+        cc_toolchain = cc_toolchain,
+        requested_features = ctx.features,
+        unsupported_features = ctx.disabled_features,
+    )
 
-    return [DefaultInfo(files = libs)]
+    build = []
+
+    for file in libs:
+        print(file)
+        library_to_link = cc_common.create_library_to_link(
+            cc_toolchain = cc_toolchain,
+            actions = ctx.actions,
+            feature_configuration = feature_configuration,
+            static_library = file,
+        )
+        build.append(library_to_link)
+
+    build = depset(build)
+    li = cc_common.create_linker_input(
+        owner = ctx.label,
+        libraries = build,
+    )
+    lc = cc_common.create_linking_context(linker_inputs = depset([li]))
+    return [CcInfo(linking_context = lc)]
```

Comment:
cc @elliottt wanted to flag that there were some changes to the build-ruby.bzl script here.